src/api/public/apidocs-new/paths/person_login.yaml

```diff
@@ -0,0 +1,37 @@
+get:
+  summary: Get details about a person
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: path
+      name: login
```

Comment:
You could refer to [login.yaml](https://github.com/openSUSE/open-build-service/blob/master/src/api/public/apidocs-new/components/parameters/login.yaml) instead.